src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,101 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+import { getNumbers } from '../../utils';
+
+type PaginationButtonProps = {
+  dataCy: string;
+  label: string;
+  active: boolean;
+  disabled: boolean;
+  onClick: () => void;
+  ariaDisabled?: boolean;
+};
+
+const PaginationButton: React.FC<PaginationButtonProps> = ({
+  active,
+  dataCy,
+  label,
+  disabled,
+  onClick,
+  ariaDisabled = true, // Set the default value to true
```

Comment:
```suggestion
  ariaDisabled = true,
```
The code says the same :)